### PR TITLE
Enrich

### PR DESCRIPTION
**Pull request by Leuchak**
There are a lot of changes, but it's hard to keep track of which pull requests are going to be included by this merge.

There's the navigation branch, tpv urdf, camera configuration, docker configuration for simulation. There may be other changes.

### CODE REVIEW

File: markhor_bringup/launch/cameras/markhor_all_cameras.launch [hunk 1]
```diff
@@ -1,5 +1,4 @@
 <launch>
-     <include file="$(find markhor_bringup)/launch/cameras/markhor_camera00_cam.launch" />
```
Comment (benmalenfant) [inline markhor_bringup/launch/cameras/markhor_all_cameras.launch:2]: Seems like a mistake to remove camera0 permanently

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,71 @@
+version: '3.9'
```
Comment (benmalenfant) [inline docker-compose.yml:1]: Why is there docker stuff in this branch?

**Review by benmalenfant — APPROVED** [review]
Seems ok, lots of stuff related to navigation

File: docker-compose.yml (hunk 2 above)
Comment (Leuchak) [reply docker-compose.yml:1]: @SimonR99 insisted and all branches written during the competition have been merged in enrich.

File: markhor_bringup/launch/cameras/markhor_all_cameras.launch (hunk 1 above)
Comment (Leuchak) [reply markhor_bringup/launch/cameras/markhor_all_cameras.launch:2]: Adding/removing cameras should be handled differently anyway. I'll leave the removal. The aim is to bring master up to date and ensure that any future changes are easy to review. We've already got 2 branches based on enrich, and it's rapidly becoming difficult to review.

**Review by SimonR99 — APPROVED** [review]

**Pull request merged**